### DISCUSSION TITLE
Update-2022-12-25 Happy Holidays

### DISCUSSION THREAD

**Discussion by kaveh808** [kaveh808/kons-9#202] [Announcements]
Hello everyone,

It is still Christmas Day here in Honolulu. Though the day has ended in the Mainland (as Hawaii residents refer to the rest of the US) and Europe, where most of our team is located, I still wanted to wish everyone Happy Holidays and a great new year.

It has been exciting to see kons-9 progress from its early days of CCL on MacOS to a cross-platform SBCL system.

Goals for this coming year include moving to Vulkan/Metal and (always a priority for me) easier installation and on-boarding of new users. And of course looking forward to having our users adopt the system as a software platform or use in their areas of interest.

So thank you for being part of the team, and looking forward with optimism to a brand new year of possibilities!

Regards,

Kaveh

**Comment by aykaramba** [comment]
Right back at ya Kaveh and team.  Having a 3D package available in the Lisp ecosystem is a hugely important project, one that will inspire us to think and dream about what can be.

**Comment by bigos** [comment]
I experiment with using Lisp for Gtk and Cairo GUI. Various solutions have different combinations of disadvantages. I hope your project becomes the platform I was looking for.

**Reply by kaveh808** [reply]
Currently the very rudimentary GUI is written in OpenGL. We hope to transition it to Vulkan once we move to that architecture.

**Reply by bigos** [reply]
I have a very basic Cairo example that is not kons-9 related. Gtk4 promises Vulkan support. Should I play with rendering my canvas in Vulkan?

**Reply by kaveh808** [reply]
Do what best suits your needs. We will revisit the GUI once we merge Vulkan into the repo.

**Reply by bigos** [reply]
It's a good idea. Let's see where it takes me.
